src/mmrelay/plugin_loader.py

```
@@ -167,6 +173,195 @@ def _temp_sys_path(path: str):
+    allow_untrusted = bool(
+        _get_security_settings().get("allow_untrusted_dependencies", False)
+    )
+    safe: List[str] = []
+    flagged: List[str] = []
+
+    for token in requirements:
+        if not token or token.startswith("#"):
+            continue
+
+        normalized = token.strip()
+        lowered = normalized.lower()
```

Ruff (0.14.3)

Local variable `lowered` is assigned to but never used

Remove assignment to unused variable `lowered`

(F841)